### PR TITLE
Add tooltip for the CCLOW map

### PR DESCRIPTION
**Pull request by aabdaab**
* Tooltip appears on bubble click
* Set stroke for clicked circle
* Tooltip is clickable - set a link inside
* Style the tooltip
* Placeholder data

image

### CODE REVIEW

**Comment by simaob** [comment]
Hey! =) Is it too confusing if it appears on hover? On the ToDo link you could point to the country page directly: https://laws-pathways-staging.vizzuality.com/cclow/geographies/143

**Comment by aabdaab** [comment]
On the design it appears only on click: https://projects.invisionapp.com/d/main#/console/18784734/391067164/inspect

**Comment by aabdaab** [comment]
And the placeholder stuff is there cause there's no data, so once it's hooked up it should be pointing to the country of the marker

File: app/assets/stylesheets/cclow/_world-map.scss [hunk 1]
```diff
@@ -8,6 +11,58 @@
     background-color: $grey-light;
   }
 
+  &__tooltip {
+    background: $white !important;
+    color: $storm-gray !important;
```
Comment (aabdaab) [inline app/assets/stylesheets/cclow/_world-map.scss]: both needed to be `!important` to override the 3rd party library

File: app/javascript/components/world-map/WorldMap.js [hunk 2]
```diff
@@ -20,6 +21,26 @@ const PetersGall = geoCylindricalEqualArea().parallel(45);
...
+        <div className="world-map__tooltip-row">
+          <p className="world-map__tooltip-text">Greenhouse gas emissions (% of global emissions)</p>
+          <p className="world-map__tooltip-number">{emissions}</p>
+        </div>
+        <a href="https://todo-link-to-that-country" className="world-map__tooltip-link">{link}</a>
+      </div>
+    );
+  };
+
+  const [content, setTooltipContent] = useState('');
```
Comment (tsubik) [inline app/javascript/components/world-map/WorldMap.js]: I think I will move that into that reducer, we should either use reducer or state I guess?

**Pull request merged**